teku/src/main/java/tech/pegasys/teku/cli/options/MetricsOptions.java

```diff
@@ -108,6 +108,17 @@ public class MetricsOptions {
       arity = "0..1")
   private boolean tickPerformanceEnabled = MetricsConfig.DEFAULT_TICK_PERFORMANCE_ENABLED;
 
+  @Option(
+      names = {"--Xmetrics-block-production-timing-tracking-enabled"},
```

Comment:
Though it's hidden flag, I think we may need to add some clarity  on which "metric" is logs only, and which one is real metric and could be used in Prometheus and co

Reply:
You're right. I'm thinking of producing some actual metrics for it so we can justify the "metrics" here